repositories/MerchStoreRepository.ts

```diff
@@ -80,8 +95,7 @@ export class MerchItemOptionRepository extends BaseRepository<MerchandiseItemOpt
     await qb
       .update()
       .set({ discountPercentage })
-      .where(`item IN ${qb.subQuery()
-        .select('merch.uuid')
+      .where(`item IN ${qb.subQuery().select('merch.uuid')
         .from(MerchandiseItemModel, 'merch')
```

Comment:
Nit: reverting this change. I usually indent subqueries like this so it's easier to read.
```suggestion
      .where(`item IN ${qb.subQuery()
        .select('merch.uuid')
        .from(MerchandiseItemModel, 'merch')
```